P/POMDPSolve/build_tarballs.jl

```diff
@@ -0,0 +1,56 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "POMDPSolve"
+version = v"5.4.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/JuliaPOMDP/pomdp-solve.git", "9e1e9987e459732c4b435387a346515afcab4e01")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd pomdp-solve/
+
+autoreconf -f -i
+
+./configure --prefix=${prefix} --exec_prefix=${prefix} --build=${MACHTYPE} --host=${target}
+
+make -j${nprocs} CC=${CC} CXX=${CXX} LINKER=${CXX} CFLAGS=${CFLAGS} CXXFLAGS=${CXXFLAGS} LDFLAGS=${LDFLAGS}
```

Comment:
I don't think you actually need to specify all the flags explicitly like this on the command, they should be picked up from the environment variables by default. 
```suggestion
make -j${nprocs}
```